backend/src/Book/Infrastructure/BookImporter.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Book\Infrastructure;
+
+use App\Book\Application\StoreBook;
+use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+
+class BookImporter
+{
+    public function __construct(
+        private readonly JsonSchemaValidator $jsonSchemaValidator,
+        private readonly StoreBook $storeBook,
+        private readonly FileHandler $fileHandler
+    ) {
+    }
+
+    public function import(string $fileName): void
+    {
+        $filePath = realpath(\dirname(__DIR__)."/../../public/{$fileName}");
+        $file = fopen($filePath, 'r');
+        
+        // create a copy of the original file
+        if (!file_exists($filePath . ".copy")) {
+            copy($filePath, $filePath . ".copy");
+        }
+        
+        [$headers, $data, $dataLen] = $this->fileHandler->csvToArray($file, $filePath);
+        
+        fclose($file);
+    
+        for ($i = 0; $i < $dataLen; $i++) {
+            if ($this->validateRowData($data, $i)) {
+                $this->storeRowData($data, $i);
+            }
+            else {
+                echo "\033[31m Errore trovato sul file csv\n";
+                echo "\033[31m Le prime {$i} righe del file csv sono state caricate correttamente\n";
+                echo "\033[31m Correggere la prima riga del file csv e rilanciare lo script\n \033[0m";
```

Comment:
**requested-change:** sostituire con un logger o ritornare un oggetto che il comando prenderà in carico per mostrare l'errore a console